deploy.pl

```diff
@@ -36,15 +37,33 @@
 
 my $ua = Mojo::UserAgent->new;
 
+my $application_id = $marathon_json->{id};
+
 my $app_url = URI->new("$marathon_apps_url/".uri_escape($marathon_json->{id}))->canonical->as_string();
+
 my $res = $ua->put($app_url => json => $marathon_json)->res();
 
 if ($res->code != 200 && $res->code != 201) {
     die $res->to_string();
 }
 
+my $deployment_url = URI->new("$marathon_api_url/deployments")->canonical->as_string();
```

Comment:
Why `canonical`? I'd rather *canonicalize* when introducing `$marathon_api_url` variable and then reuse the canonical form in all derived variables.